org.scala-ide.sdt.core.tests/src/org/scalaide/core/testsetup/SDTTestUtils.scala

```diff
@@ -39,10 +39,11 @@ import org.eclipse.jdt.launching.JavaRuntime
 import org.scalaide.core.compiler.IScalaPresentationCompiler
 import org.scalaide.core.internal.project.ScalaProject
 
-/** Utility functions for setting up test projects.
+/**
+ * Utility functions for setting up test projects.
  *
  */
-object SDTTestUtils extends HasLogger {
+trait SDTTestUtils extends HasLogger {
```

Comment:
yea, I revert this change and think how to do this seamless in post 4.1. No
need to delay solution delivery. And there is a need to educate as you
mentioned :).

2015-05-18 10:42 GMT+02:00 Simon Schäfer <notifications@github.com>:

> In
> org.scala-ide.sdt.core.tests/src/org/scalaide/core/testsetup/SDTTestUtils.scala
> <https://github.com/scala-ide/scala-ide/pull/948#discussion_r30489665>:
>
> >   *
> >   */
> > -object SDTTestUtils extends HasLogger {
> > +trait SDTTestUtils extends HasLogger {
>
> Ok. Do you want to have this now or can we move refactorings to post 4.1?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scala-ide/scala-ide/pull/948/files#r30489665>.
>